custom_components/weenect/sensor.py

```diff
@@ -120,11 +194,33 @@ def async_add_sensors(
 class WeenectSensor(WeenectEntity, SensorEntity):
     """weenect sensor for general information."""
 
+    def _get_call_available(self) -> int:
+        """Return remaining call time."""
+        if (
+            "call_usage" in self.coordinator.data[self.id]
+            and "call_max_threshold" in self.coordinator.data[self.id]
+        ):
+            return (
+                self.coordinator.data[self.id]["call_max_threshold"]
+                - self.coordinator.data[self.id]["call_usage"]
+            )
```

Comment:
```suggestion
            return int(
                self.coordinator.data[self.id]["call_max_threshold"]
                - self.coordinator.data[self.id]["call_usage"]
            )
```
Not sure if this works